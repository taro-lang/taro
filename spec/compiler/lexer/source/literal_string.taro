var string : String = "simple string"