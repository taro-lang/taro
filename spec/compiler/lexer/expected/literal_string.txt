Identifier (var) line: 0, length: 3, position: (0,2)
Space ( ) line: 0, length: 1, position: (3,3)
Identifier (string) line: 0, length: 6, position: (4,9)
Space ( ) line: 0, length: 1, position: (10,10)
Space ( ) line: 0, length: 1, position: (12,12)
Identifier (String) line: 0, length: 6, position: (13,18)
Space ( ) line: 0, length: 1, position: (19,19)
Assign (=) line: 0, length: 1, position: (20,20)
Space ( ) line: 0, length: 1, position: (21,21)
Identifier (simple) line: 0, length: 6, position: (23,28)
Space ( ) line: 0, length: 1, position: (29,29)
Identifier (string) line: 0, length: 6, position: (30,35)